### ISSUE TITLE
maya2glTF.mel invalid encoding

### ISSUE THREAD

**Issue by robertlong** [Ziriax/Maya2glTF#32]
The `<FileUpdate>` MSBuild task for updating the version in the mel script uses UTF-8 with BOM encoding when I believe Maya expects UTF8.

I see this error when attempting to load the MEL script:
```
source "C:/Users/robertlong/Documents/maya/2018/scripts/maya2glTF_UI.mel";
// Error: ï»¿proc string _quoted(string $text)
 // 
// Error: "C:/Users/robertlong/Documents/maya/2018/scripts/maya2glTF_UI.mel" line 1.2: Syntax error // 
```

I can't make a PR right now because of the LFS quota (see #31) but here's the fix.

In Maya2glTF.vcxproj change this:
```
<FileUpdate Files="$(ProjectDir)\maya\scripts\maya2glTF_UI.mel" Multiline="true" Singleline="false" Regex="\$maya2glTF_version\s*\=\s*&quot;(.+)&quot;" ReplacementText="$maya2glTF_version = &quot;$(GIT_Tag) $(GIT_Hash)&quot;" />
```

to:

```
 <FileUpdate Files="$(ProjectDir)\maya\scripts\maya2glTF_UI.mel" Multiline="true" Singleline="false" Regex="\$maya2glTF_version\s*\=\s*&quot;(.+)&quot;" ReplacementText="$maya2glTF_version = &quot;$(GIT_Tag) $(GIT_Hash)&quot;" Encoding="utf-8-without-bom" />
```

**Comment by Ziriax** [comment]
Great! This was really driving me mad, but I didn't find a workaround for it. Thanks for this.

BTW: I am going to transfer this repository to the WonderMediaProduction organisation. This started as a personal project, but my employer embraces Maya2glTF, and we will be using glTF as the main asset type in all our future productions. This should also solve the quota problems. However, I'm not sure what implications this has for you. Are you okay with this transfer? It will remain a public repository with the same license model.

**Comment by Ziriax** [comment]
Fixed by 9ce1b126e011c8c4eca5abfa8234e5a668934598.

Thanks @robertlong
